packages/dita-example-sx-integration/src/configureSxModule.js

```diff
@@ -42,19 +43,21 @@ export default function configureSxModule(sxModule) {
 	// Needed for bookmap implementation
 	configureProperties(
 		sxModule,
-		'self::*[fonto:dita-class(., "topic/topic")][not(parent::*[fonto:dita-class(., "topic/topic")])]',
+		xq`self::*[fonto:dita-class(., "topic/topic")][not(parent::*[fonto:dita-class(., "topic/topic")])]`,
 		{
-			titleQuery: `
+			titleQuery: xq`
 			let $refNodeName := fonto:hierarchy-source-node(fonto:current-hierarchy-node-id())/name(),
 			$title := ./*[fonto:dita-class(., "topic/title")]//text()[not(ancestor::*[name() = ("sort-at", "draft-comment", "foreign", "unknown", "required-cleanup", "image")])]/string() => string-join()
 
 			return
 				if(not($refNodeName) or not(bookmap:retrieve-element-label($refNodeName)))
 				then $title
 				else string-join(upper-case(bookmap:retrieve-element-label($refNodeName)) || ": " || $title)`,
-			blockHeaderLeft: [
-				createLabelQueryWidget('""', {
-					prefixQuery: `
+			blockHeaderLeft: 
+			// TODO: fix createLabelQueryWidget
+			[
+				createLabelQueryWidget(`""`, {
+					prefixQuery: xq`
```

Comment:
So the label query itself does not work, but the prefix part does? Maybe it's better to just leave a descriptive TODO like 
```
// TODO: Upgrade the createLabelQueryWidget when it accepts XQuery template tags in 7.18
```
and leave it alone for now. This code will be transfered to partners when they start with a new dita editor, so the TODO should also be actionable to them